telemetry/prometheus/src/test/scala/io/buoyant/telemetry/prometheus/PrometheusTelemeterInitializerTest.scala

```diff
@@ -11,6 +11,7 @@ class PrometheusTelemeterInitializerTest extends FunSuite {
   test("io.l5d.prometheus telemeter loads") {
     val yaml =
       """|kind: io.l5d.prometheus
+         |path: /metrics
```

Comment:
Very minor, but: would like to see a test that the config entry without a path configuration uses the correct default path.